proto/resources/external/exposurenotification/diagnosis_key_batch.proto

```diff
@@ -15,14 +17,13 @@ message DiagnosisKey {
   ReportType reportType = 7; // set by backend
   // Number of days elapsed between symptom onset and the TEK being used.
   // E.g. 2 means TEK is 2 days after onset of symptoms.
-  uint32 days_since_onset_of_symptoms = 8;
+  sint32 days_since_onset_of_symptoms = 8;
```

Comment:
Just a comment but `sint32` makes no sense (although no difference) here.

Reply:
Since we don't create these files there is nothing I can do at this moment. 
But we might be able to change it here: https://github.com/corona-warn-app/cwa-protocol-buffers/tree/main/src/proto/external/exposurenotification